core/granite/commons/systemRequirements.ts

```diff
@@ -0,0 +1,11 @@
+import { GB } from "./sizeUtils";
+
+export interface SystemRequirements {
+  reservedSystemMemory: number;
+  reservedGraphicsMemory: number;
+}
+
+export const SYSTEM_REQUIREMENTS: SystemRequirements = {
+  reservedSystemMemory: 6 * GB,
+  reservedGraphicsMemory: 0 * GB,
```

Comment:
Note i set this to 0 for now so medium end discrete cards still recommend the large model. we may need to tweak this value.